sn_client/src/file_apis.rs

```diff
@@ -170,8 +170,10 @@ impl Files {
         let payment = payment_proofs.get(&address.name().0).cloned();
         // TODO: re-enable requirement to always provide payment proof
         //.ok_or(super::Error::MissingPaymentProof(address))?;
-
-        self.client.store_chunk(chunk, payment).await?;
+        let closest_peers = self.client.client_get_closest_peers_from_local().await?;
```

Comment:
If we do this all the time, when/how do we populate enough info from the network? 

What happens if this data is stale?

Reply:
> If we do this all the time, when/how do we populate enough info from the network?

Here only the result from the `libp2p::get_closest_peers` got trimmed for the half-range result.
And it is only `libp2p::get_closest_peers` will populate network knowledge.
So no affect to the network info pupulation.

> What happens if this data is stale?